scalafix-tests/output/src/main/scala/test/RemoveUnusedTerms.scala

```diff
@@ -0,0 +1,13 @@
+package test
+
+object RemoveUnusedTerms {
+
+  def foo {
+    
```

Comment:
You may want to remove the leading spaces including the newline (like I did in my presentation from yesterday)

Reply:
We definitely need a ctx.removeLeading or something

Reply:
Yes, I suggested at the spree that we leave the leading space unchanged until we add better utilities to handle this.